components/Paper/Upload/api/useEffectGetSuggestedTags.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  emptyFncWithMsg,
+  isNullOrUndefined,
+  nullthrows,
+} from "../../../../config/utils/nullchecks";
```

Comment:
BTW, we implemented some config change recently that allows you to use "~" to refer to the root dir. You can do `~/config/utils/nullchecks` instead